product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Connecting to Your Cluster"
+---
+You can connect to your cluster using [`psql`](http://postgresguide.com/utilities/psql.html), the terminal-based client for Postgres, or another client.
+
+To connect to your cluster using `psql`:  
+1. Go to the **Clusters** page.
+2. Select the name of your cluster. 
+3. On the **Overview** tab, select the copy icon to the right of the **Quick Connect** field to copy the command for connecting to your cluster using `psql`. 
```

Comment:
Should we include an image here?

Reply:
@ba-edb I don't think it is necessary, but am happy to discuss it. These are the guidelines I generally go by: https://enterprisedb.atlassian.net/wiki/spaces/~933724608/pages/2727903246/Screenshot+Policy.